main.go

```diff
@@ -103,13 +103,13 @@ func main() {
 
 	flag.IntVar(&awsClusterConcurrency,
 		"awscluster-concurrency",
-		2,
+		10,
```

Comment:
Can we bump AWSCluster down to 5 for now?

Reply:
Sure